pkg/utils/compareContainerId.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+func CompareContainerId(containerIdBefore [9]string, podNs string, podLabels string, client *kubernetes.Clientset) bool {
```

Comment:
Add comment over the function

Reply:
yes one pod contains 3 containerID we can have 1 pod (target pod) or 3 pods (csp pod)